assets/sass/cds/_sections.scss

```diff
@@ -1469,3 +1430,75 @@ article.post {
     margin-bottom: 0;
   }
 }
+.work-with-us--apply-now {
+  width: 65%;
```

Comment:
This doesn't seem like just a move, are they changes here expected for the job page?

Reply:
Yes: https://www.figma.com/file/LW1JsT0h3IY2trMHdIVDQc/CDS-Website-Revamp-(Work-in-Progress)?node-id=132%3A8131